### ISSUE TITLE
Labels/ticks never appear

### ISSUE THREAD

**Issue by APB9785** [mindok/contex#52]
I am using code which is basically copy/pasted from the example at contex-charts.org
```
defp plot_chart(record_list) do
    options = [
      mapping: %{category_col: "date", value_cols: ["load"]},
      type: :stacked,
      data_labels: false,
      orientation: :vertical,
      colour_palette: ["ff9838", "fdae53", "fbc26f", "fad48e", "fbe5af", "fff5d1"]
    ]

    record_list
    |> Contex.Dataset.new(["date", "load"])
    |> Contex.Plot.new(Contex.BarChart, 600, 400, options)
    |> Contex.Plot.axis_labels("", "")
    |> Contex.Plot.plot_options(%{})
    |> Contex.Plot.titles("My first plot", "With a fancy subtitle")
  end
```
And everything shows up fine, except there are no labels on the axis.  I played around with the options, but couldn't find any documentation about this.  The documentation even says it should "just work" with minimal tinkering.

Here is the result I am getting:
![Screen Shot 2021-06-23 at 12 49 57 PM](https://user-images.githubusercontent.com/74077809/123144749-a1e3c400-d421-11eb-8086-3fd6344b8e77.png)

Also the color palette is totally different, which is strange because I am pulling those values straight from the example on contex-charts.org

EDIT:  `data_labels: false` also doesn't appear to be taking effect, as you can see in the pic.

**Comment by mindok** [comment]
Are you getting the dependency from hex or github at the moment? Try setting the dependency to point to github and let us know how you go:

```elixir
{:contex, git: "https://github.com/mindok/contex"},
```

You will need to update your dependencies once you have made the edit:
```
mix deps.unlock contex
mix deps.update contex
```

**Comment by gainhad** [comment]
Had the same issue. Pointing the dependency at GitHub fixed this for me. Thanks!

**Comment by mindok** [comment]
Fixed in 0.4.0
